pkg/kubernetes/announcement_handlers_test.go

```diff
@@ -0,0 +1,107 @@
+package kubernetes
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+
+	"github.com/google/uuid"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/fake"
+
+	"github.com/openservicemesh/osm/pkg/announcements"
+	"github.com/openservicemesh/osm/pkg/constants"
+	"github.com/openservicemesh/osm/pkg/kubernetes/events"
+)
+
+var _ = Describe("Test Announcement Handlers", func() {
```

Comment:
Not a blocker, but I'd suggest using Go's testing lib instead of Ginkgo based on decisions we made previously to write new tests using Go's testing lib instead.